packages/grant-explorer/src/features/common/__tests__/ErrorModal.test.tsx

```diff
@@ -77,4 +77,32 @@ describe.skip("<ErrorModal />", () => {
     expect(setIsOpenFn).toBeCalledTimes(1);
     expect(setIsOpenFn).toBeCalledWith(false);
   });
+
+  it.skip("should close the modal if close on background click is enabled and background is clicked", () => {
```

Comment:
Should this stay skipped?

Reply:
I wasn't able to test this part of the modal. The funcitonality that it's testing is basically that we pass the "close on background click" prop correctly from our component to the headlessui component. I would welcome a better way to test just that, instead of testing the implementation detail of the actual headless ui component, which we have no business (lol) with.

Reply:
Ok, just double-checking because I've sometimes marked tests as skip/only during development then forgotten to un-mark them before pushing.